docs/installation.rst

```diff
@@ -197,6 +197,30 @@ and commands to make fuse work for using the mount command.
      sysctl vfs.usermount=1
     
 
+Windows (MSYS2)
++++++++++++++++
+
+Download MSYS from https://msys2.github.io/ and follow install instructions. This guide assumes 64bit version.
+
+Run `pacman -S mingw-w64-x86_64-python3 git mingw-w64-x86_64-lz4 mingw-w64-x86_64-python3-pip mingw-w64-x86_64-cython mingw-w64-x86_64-gcc mingw-w64-x86_64-python3-pytest` to install required packages.
```

Comment:
This could be something like:

Install the dependencies:

    pacman -S mingw-w64-x86_64-python3 git mingw-w64-x86_64-lz4 mingw-w64-x86_64-python3-pip \
              mingw-w64-x86_64-cython mingw-w64-x86_64-gcc mingw-w64-x86_64-python3-pytest